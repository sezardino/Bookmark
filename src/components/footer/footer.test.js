import React from 'react';
import {shallow} from 'enzyme';

import Footer from './footer';

describe('Component Footer', () => {
	const component = shallow(<Footer />);

	it('Component Footer rendered correctly', () => {
		expect(component.debug()).toMatchSnapshot();
	});
});
