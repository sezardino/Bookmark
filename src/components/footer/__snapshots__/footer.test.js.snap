// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Footer Component Footer rendered correctly 1`] = `
"<footer>
  <section className=\\"subscribe\\">
    <h2 className=\\"hidden\\">
      Subscribe on updates
    </h2>
    <div className=\\"container subscribe__container\\">
      <p className=\\"subscribe__count\\">
        <span className=\\"counter\\">
          35.000
        </span>
         + already joined
      </p>
      <h3 className=\\"subscribe__title\\">
        Stay up-to-date with what we&#39;re doing
      </h3>
      <Form />
    </div>
  </section>
  <section className=\\"footer\\">
    <h2 className=\\"hidden\\">
      More information you can find here
    </h2>
    <div className=\\"container footer__container\\">
      <p className=\\"logo footer__logo\\">
        <img src=\\"./img/logo-footer.svg\\" alt=\\"company logo\\" />
      </p>
      <Nav />
    </div>
  </section>
</footer>"
`;
